nevergrad/optimization/recorded_recommendations.csv

```diff
@@ -1,219 +0,0 @@
-,v0,v1,v2,v3,v4,v5,v6,v7,v8,v9,v10,v11,v12,v13,v14,v15
```

Comment:
the file need to be commited again
also, a faster way is to only remove the offending lines